dev/VSIX/ProjectTemplates/Desktop/CppWinRT/PackagedApp/BlankApp/pch.h

```diff
@@ -21,4 +21,5 @@
 #include <winrt/Microsoft.UI.Xaml.Media.h>
 #include <winrt/Microsoft.UI.Xaml.Navigation.h>
 #include <winrt/Microsoft.UI.Xaml.Shapes.h>
-#include <winrt/Microsoft.UI.Dispatching.h>
\ No newline at end of file
+#include <winrt/Microsoft.UI.Dispatching.h>
+#include <wil/cppwinrt.h>
```

Comment:
IIRC, this header needs to come _before_ any C++/WinRT headers are included.  See https://github.com/microsoft/wil/blob/a762808b1cd32b208b6f7ecd3a2f3ed32b9c3815/include/wil/cppwinrt.h#L78 - samples using this template should have failed to compile with this ordering.